spring/spring4/pom.xml

```diff
@@ -183,6 +193,18 @@
             <artifactId>commons-dbcp</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>javax.cache</groupId>
+            <artifactId>cache-api</artifactId>
+            <version>${version.javax.cache.cache-api}</version>
```

Comment:
Version might be removed - it is already specified in DependencyManagement section of `bom/pom.xml`